gradoop-store/gradoop-accumulo/src/test/java/org/gradoop/flink/io/impl/accumulo/IOBasicTest.java

```diff
@@ -95,9 +92,8 @@ public void test01_read() throws Exception {
 
   @Test
   public void test02_write() throws Exception {
```

Comment:
Please do not use the deprecated `EPGMDatabase` class and its `writeTo()` method. Create a Sink and use it for writing the graph from loader